src/main/java/com/autotune/analyzer/recommendations/RecommendationConstants.java

```diff
@@ -500,6 +524,11 @@ public static final class NotificationCodes {
         public static final int SECTION_CRITICAL_SUBSECTION_DATA_SUBSYSTEM_MEMORY_START = 524000;
         public static final int CRITICAL_MEMORY_REQUEST_NOT_SET = 524001;
         public static final int CRITICAL_MEMORY_LIMIT_NOT_SET = 524002;
+        public static final int ERROR_NO_RECOMMENDED_REQUEST_FOR_LIMIT_CALCULATION = 225001;
```

Comment:
Need to add these new notification codes in the design doc as well.
For reference: https://github.com/kruize/autotune/blob/master/design/NotificationCodes.md